content/en/docs/started/workstation/getting-started-multipass.md

```diff
@@ -42,55 +33,48 @@ it, and then set up Kubeflow.
     microk8s.status --wait-ready
     ```
 
-3. Having installed MicroK8s, you can now enable common services on your
-   MicroK8s deployment. To do that, run the following command:
+3. Having installed MicroK8s, you can now enable common services on your MicroK8s deployment. To do that, run the following command:
 
     ```
     microk8s.enable dns dashboard storage
     ```
 
-    **Optional:** To enable NVIDIA GPU hardware support, also run
-    `MicroK8s.enable gpu`.
+    **Optional:** To enable NVIDIA GPU hardware support, also run `MicroK8s.enable gpu`.
```

Comment:
@RFMVasconcelos Can you please check if it's `microK8s.enable gpu` (lower caps)? if this works too, no worries

Reply:
Oh yeah, lower is better!